backends/dpdk/constants.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef BACKENDS_DPDK_CONSTANTS_H_
+#define BACKENDS_DPDK_CONSTANTS_H_
+
+/* Unique handle for action and table */
+#define TABLE_HANDLE_PREFIX 0x00010000
```

Comment:
I think that the preferred C++ way is to use a const variable, but this will work too.

Reply:
Fixed